app/static/js/event_list.js

```diff
@@ -1,5 +1,34 @@
 
 
+$(document).ready(function(){
+
+  // $("#sendEmail").click(function(){
+  //   console.log($("#emailSender").val())
+  //   console.log($("#message").val())
+  //   // msg = $("#message").val().replace(" ",'%20')
+  //   //
+  //   // console.log($("#subject").val())
+  //   // subject = self.emailInfo['subject'].replace(" ",'%20')
+  //
+  //   // window.location.href = "mainto:"+$("#emailSender").val()+"?Subject="+$("#subject").val()
+  //   window.location.href = "mailto:"+$("#emailSender").val()
+  //   // window.location.href = `mailto:${encodeURIComponent(("#emailSender").val())}?subject=+${("#subject").val()}`
+  //   // `{encodeURIComponent($("#subject").val())}&body=${encodeURIComponent($("#message").val())}`;
+  // })
+
+});
```

Comment:
Let's get rid of this code if it isn't needed. If it might be needed, add some comments to indicate the use case

Reply:
still relevant